django/utils/_os.py

```diff
@@ -5,20 +5,19 @@
 from django.core.exceptions import SuspiciousFileOperation
 from django.utils.encoding import force_text
 
+# For backwards-compatibility in Django 2.0
```

Comment:
I created https://code.djangoproject.com/ticket/27753 to track the eventual removal of these functions.